CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ parent: README
 nav_order: 1
 ---# Changelog
 
+## [2.1.11] - 2021-04-09
+- Add support for Kafka Topic Tags
+- Fix Project name updates
```

Comment:
Use of caps looks weird here, I'd expect "Add support for Kafka topic tags" and "Fix project name updates"

Reply:
indeed